### ISSUE TITLE
Installation problems STM32CubeIDE (Eclipse 4.23)

### ISSUE THREAD

**Issue by tusker-tools** [grosenberg/IndentGuide#21]
Hallo,
I just tried to install this plugin in STM32CubeIde v1.11.2 which is based on Eclipse 4.23.

Unfortunately the installation process ends with following error message:
```
Cannot complete the install because one or more required items could not be found.
  Software being installed: IndentGuide 2.0.0.20230220-0508 (net.certiv.tools.indentguide.feature.feature.group 2.0.0.20230220-0508)
  Missing requirement: IndentGuide 2.0.0.20230220-0508 (net.certiv.tools.indentguide.feature.feature.group 2.0.0.20230220-0508) requires 'org.eclipse.equinox.p2.iu; org.eclipse.ui [3.201.200,4.0.0)' but it could not be found
```

Checking the installed plugins I can seee that I have following version installed:
`	Eclipse.org	Eclipse UI	3.201.0.v20220124-1108	org.eclipse.ui`

Is there any possibility how I can get the Indent Guide Plugin running anyway?

Thanks in advance!

**Comment by grosenberg** [comment]
Can the IDE not be updated? Their website is showing a newer version.

**Comment by tusker-tools** [comment]
That's a good point. I just updated to v1.12.0. Unfortunately the plugin installation still fails. The org.eclipse.ui version seems to be still the same.

**Comment by grosenberg** [comment]
But can you update the IDE -- does `Help->Check for Updates` exist?

**Comment by tusker-tools** [comment]
Yes, as I mentioned above an update was available and I updatet the IDE from version 1.11.2 to the new version 1.12.0 (by using "Help-> Check for Updates" Menu)

Or have I misunderstood your question?

**Comment by grosenberg** [comment]
Seems they have the IDE locked down.  

A development version of IndentGuide that installs and runs well on standard Eclipse 4.23 will simply not install on STM32CubeIde v1.12.

Not anything that can be done to fix this from this end.

[net.certiv.tools.indentguide.site-2.1.0-SNAPSHOT.zip](https://github.com/grosenberg/IndentGuide/files/10947722/net.certiv.tools.indentguide.site-2.1.0-SNAPSHOT.zip)

**Comment by tusker-tools** [comment]
Finally, for the newly released STM32CubeIDE v1.12.1, the plugin is working!
Just tried it for curiosity and bump!, the installation was successful ;-)

Thanks for this useful plugin. Happy to be able to use it in the end!
